canal/handler.go

```diff
@@ -40,3 +40,24 @@ func (h *DummyEventHandler) String() string { return "DummyEventHandler" }
 func (c *Canal) SetEventHandler(h EventHandler) {
 	c.eventHandler = h
 }
+
+// EventHandlerV2 can process event with replication.EventHeader
+type EventHandlerV2 interface {
```

Comment:
sorry for my previous comments, now I think we can change the old `EventHandler` to this interface (will review more carefully later)

Reply:
I have changed EventHandlerV2 to EventHandler